lib/chewy/type/crutch.rb

```diff
@@ -23,7 +24,11 @@ def #{name}
 
       module ClassMethods
         def crutch name, &block
-          self._crutches = _crutches.merge(name.to_sym => block)
+          if block
+            self._crutches = _crutches.merge(name.to_sym => block)
+          else
+            self._crutches_classes = _crutches_classes + [name]
```

Comment:
So I don't see any point of whole change :wink: 
As far as i can see it will only allow us to change this:
`value: -> (x, crutch) { crutch.something[x.id] }`
into: `value: -> (x, _, something) { something.else(x) }`